pkg/plugins/golang/v3/init.go

```diff
@@ -88,6 +89,9 @@ func (p *initSubcommand) BindFlags(fs *pflag.FlagSet) {
 		"license to use to boilerplate, may be one of 'apache2', 'none'")
 	fs.StringVar(&p.owner, "owner", "", "owner to add to the copyright")
 
+	fs.BoolVar(&p.useWorkspaces, "workspace", false, "if true, scaffolds apis with `go.work` workspace differentiation, "+
```

Comment:
If we move forward with flags then I think we ought to do it as multi-group option